doc/demo.rst

```diff
@@ -1788,95 +1984,114 @@ transport node:
 Transporting the transport node
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now let’s simulate what would happen if we wanted to move this transport
-node from ``alpen3`` to ``alpen1`` over the Snearkernet. (Normally, to
+Now let's simulate what would happen if we wanted to move this transport
+node from ``alpenhost3`` to ``alpenhost1`` over the Snearkernet. (Normally, to
 increase throughput of the Snearkernet, we would wait for the node to
-fill up, but we’re not going to wait for that in this demo.)
+fill up, but we're not going to wait for that in this demo.)
 
 The first step is to deactivate the alpenhorn node to tell alpenhorn to
 stop managing it:
 
-::
+.. code:: console
+   :class: democontainer
 
    alpenhorn node deactivate transport1
 
-The daemon on ``alpen3`` will notice this, and stop updating the
+The daemon on ``alpenhost3`` will notice this, and stop updating the
 transport node:
 
-::
+.. code:: console
+   :class: demohost
+
+   alpenhost3-1  | Mar 09 04:10:07 INFO >> [MainThread] Node "transport1" no longer available.
+   alpenhost3-1  | Mar 09 04:10:07 INFO >> [MainThread] Group "transport_group" no longer available.
 
-   alpen3-1  | Mar 09 04:10:07 INFO >> [MainThread] Node "transport1" no longer available.
-   alpen3-1  | Mar 09 04:10:07 INFO >> [MainThread] Group "transport_group" no longer available.
+.. hint::
+   A StorageGroup is only available to a daemon if at least one of its nodes is
+   available.  When we deactivate the ``transport1`` node, resulting it it no
+   longer being available, the ``transport_group`` also becomes unavailable because
+   there are no other active nodes on ``alpenhost3``.
```

Comment:
> -> Does transport group become unavailable after transport1 is deactivated only because transport 1 is the only node in the group?

A good question.  Hopefully this hint answers it.